src/utilities/graph-util.js

```diff
@@ -175,6 +175,56 @@ class GraphUtils {
   static isEqual(prevNode: any, newNode: any) {
     return fastDeepEqual(prevNode, newNode);
   }
+
+  static findNodesWithinArea(start: IPoint, end: IPoint, nodes: INode[]) {
+    const smallerX = start.x <= end.x ? start.x : end.x;
+    const smallerY = start.y <= end.y ? start.y : end.y;
+    const largerX = end.x > start.x ? end.x : start.x;
+    const largerY = end.y > start.y ? end.y : start.y;
+
+    // Note: it's possible for end to be smaller than start
```

Comment:
Maybe remove this comment? Assume it's meant to apply to constants above otherwise you wouldn't need to set smallerX, smallerY, etc.

Reply:
This is a note for any future developer to understand why the above was necessary. I'd prefer to leave it because the code can be a little confusing.

Reply:
Thanks. Done.